src/leaflike/user/auth.clj

```diff
@@ -1,13 +1,12 @@
 (ns leaflike.user.auth
   (:require [buddy.hashers :as hashers]
             [buddy.auth.backends.session :refer [session-backend]]
-            [leaflike.user.db :refer [get-member-auth-data]]
-            [buddy.auth :refer [authenticated?]]
-            [ring.util.response :as res]))
+            [ring.util.response :as res]
+            [buddy.auth :refer [authenticated?]]))
 
-(defn- user-auth-data
+#_(defn- user-auth-data
```

Comment:
Commented code should be treated like dead code: don't check it in, delete it with a vengeance.
If people want to look at old code, they have git.